clang/unittests/Format/NumericLiteralInfoTest.cpp

```diff
@@ -47,14 +47,21 @@ TEST_F(NumericLiteralInfoTest, IntegerLiteral) {
   // Hexadecimal.
   EXPECT_TRUE(verifyInfo(NumericLiteralInfo("0xF"), 1));
   EXPECT_TRUE(verifyInfo(NumericLiteralInfo("0XfZ"), 1, npos, npos, 3));
+
+  // C++ suffixes.
+  EXPECT_TRUE(verifyInfo(NumericLiteralInfo("1if"), npos, npos, npos, 1));
+  EXPECT_TRUE(verifyInfo(NumericLiteralInfo("123_felt_pen"), npos, npos, npos, 3));
```

Comment:
Why the new test cases for `NumericLiteralInfo`? If they test non-trivial code paths not already covered by the existing tests, please add them in a separate patch.